sdk/storage/azure-storage-file-datalake/tests/test_directory.py

```diff
@@ -95,6 +96,175 @@ def test_create_directory(self, datalake_storage_account_name, datalake_storage_
         # Assert
         self.assertTrue(created)
 
+    @DataLakePreparer()
+    def test_create_file_owner_with_directory_client(self, datalake_storage_account_name, datalake_storage_account_key):
```

Comment:
We could combine owner/group/acl into one test to save on test/recoding count :). This would apply to files as well.